docs/zh/operations/table_engines/distributed.md

```diff
@@ -54,72 +54,72 @@ Clusters are set like this:
-This works in favor of resiliency, but does not provide complete fault tolerance: a remote server might accept the connection, but might not work, or work poorly.
+配置了副本，读取操作会从每个分片里选择一个可用的副本。可配置负载平衡算法（挑选副本的方式） - 请参阅“load_balancing”设置。
+如果跟服务器的连接不可用，则在尝试短超时的重连。如果重连失败，则选择下一个副本，依此类推。如果跟所有副本的连接尝试都失败，则尝试用相同的方式再重复几次。
+该机制有利于系统韧性，但不保证完全容错：如有远程服务器可能能连接上，但无法正常工作或状况不佳。
 
-You can specify just one of the shards (in this case, query processing should be called remote, rather than distributed) or up to any number of shards. In each shard, you can specify from one to any number of replicas. You can specify a different number of replicas for each shard.
+你可以配置一个（这种情况下，查询操作更应该称为远程查询，而不是分布式查询）或任意多个分片。在每个分片中，可以配置一个或任意多个副本。不同分片可配置不同数量的副本。
 
-You can specify as many clusters as you wish in the configuration.
+可以在配置中配置任意数量的群集。
 
-To view your clusters, use the 'system.clusters' table.
+要查看群集，可使用“system.clusters”表。
 
-The Distributed engine allows working with a cluster like a local server. However, the cluster is inextensible: you must write its configuration in the server config file (even better, for all the cluster's servers).
+通过分布式引擎可以像使用本地服务器一样使用集群。但是，群集不是自动扩展的：你必须编写集群配置到服务器配置文件中（最好，给所有群集的服务器写上完整配置）。
 
-There is no support for Distributed tables that look at other Distributed tables (except in cases when a Distributed table only has one shard). As an alternative, make the Distributed table look at the "final" tables.
+不支持用分布式表查询别的分布式表（除非该表只有一个分片）。或者说，要用分布表查查询“最终”的数据表。
 
-The Distributed engine requires writing clusters to the config file. Clusters from the config file are updated on the fly, without restarting the server. If you need to send a query to an unknown set of shards and replicas each time, you don't need to create a Distributed table – use the 'remote' table function instead. See the section "Table functions".
+分布式引擎需要将集群信息写入配置文件。配置文件中的群集信息会即时更新，无需重启服务器。如果你每次是要向不确定的一组分片和副本发送查询，则不适合创建分布式表 - 而应该使用“远程”表函数。 请参阅“表函数”部分。
 
-There are two methods for writing data to a cluster:
+向集群写有数据的方法有两种：
 
-First, you can define which servers to write which data to, and perform the write directly on each shard. In other words, perform INSERT in the tables that the distributed table "looks at".
-This is the most flexible solution – you can use any sharding scheme, which could be non-trivial due to the requirements of the subject area.
-This is also the most optimal solution, since data can be written to different shards completely independently.
+一，自已指定要将哪些数据写入哪些服务器，并直接在每个分片上执行写入。换句话说，在分布式表上“查询”，在数据表上 INSERT。
+这是最灵活的解决方案 – 你可以使用任何分片方案，对于业务特性的需求，这可能是非常重要的。<!-- "which could be non-trivial due to the requirements of the subject area." Oh my god! Is this right -->
```

Comment:
@blinkov I don't quite understand what that means either : )

Reply:
I think this phrase is supposed to mean that you can have arbitrary complex business logic that decides which piece of data goes to which local table (since in this scenario it's implemented in client application and thus is not limited by declarative nature of SQL).